pod/live/models.py

```diff
@@ -100,6 +103,9 @@ class Broadcaster(models.Model):
         help_text=_(
             'Viewing this live will not be possible without this password.'),
         max_length=50, blank=True, null=True)
+    viewcount = models.IntegerField(
+        _('Number of viewers'), default=0, editable=False)
+    viewers = models.ManyToManyField(User, editable=False)
```

Comment:
le top du top serait d'avoir un many to many vers un model viewer qui aurait le user, la date et l'heure de debut et de fin (i had a dream)

Reply:
Pour pouvoir faire des statistiques ? car le but de ces champs est juste de sauvegarder temporairement l'état d'un live pour limiter les calculs ici, donc ça serait quelque chose en plus

Reply:
ok. Du coup, si on souhiates voir combien de personne ont suivi un live après coup, est-ce possible ? si oui comment ?

Reply:
Actuellement non, on est sur du temps réel uniquement, mais ça pourrait être une bonne évolution (mais ça demande encore du temps de dev)

Reply:
Du coup je clos la remarque mais je travaillerais dessus pour la prochaine version